config/routes.rb

```diff
@@ -1,8 +1,14 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  root "pages#blog"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
 
-  # Additional routes...
+  get '/users/:user_id/posts', to: 'posts#index', as: 'posts'
+  get '/users/:user_id/posts(/:id)', to: 'posts#show', as: 'post'
+
```

Comment:
- [ ] Please remove these two routes because you already added them below. you can also remove `module: :users` at the end of line 10.